packages/cli/api-importers/openapi/openapi-ir-to-fern-tests/src/__test__/__snapshots__/openapi-ir/vellum.json

```diff
@@ -4126,130 +4126,23 @@
-                    "value": {
-                      "generatedName": "DocumentsUploadRequestAddToIndexNamesItem",
-                      "schema": {
-                        "minLength": 1,
-                        "type": "string"
-                      },
-                      "groupName": [],
-                      "type": "primitive"
-                    },
-                    "description": "Optionally include the names of all indexes that you'd like this document to be included in",
-                    "groupName": [],
-                    "type": "array"
-                  },
-                  "description": "Optionally include the names of all indexes that you'd like this document to be included in",
-                  "groupName": [],
-                  "type": "nullable"
-                },
-                "groupName": [],
-                "type": "optional"
-              },
-              "type": "json"
-            }
-          },
-          {
-            "key": "external_id",
-            "schema": {
-              "value": {
-                "generatedName": "documentsUploadRequestExternalId",
-                "value": {
-                  "generatedName": "DocumentsUploadRequestExternalId",
-                  "value": {
-                    "generatedName": "DocumentsUploadRequestExternalId",
-                    "schema": {
-                      "minLength": 1,
-                      "type": "string"
-                    },
-                    "description": "Optionally include an external ID for this document. This is useful if you want to re-upload the same document later when its contents change and would like it to be re-indexed.",
-                    "groupName": [],
-                    "type": "primitive"
-                  },
-                  "groupName": [],
-                  "description": "Optionally include an external ID for this document. This is useful if you want to re-upload the same document later when its contents change and would like it to be re-indexed.",
-                  "type": "nullable"
-                },
-                "groupName": [],
-                "type": "optional"
-              },
-              "type": "json"
-            }
-          },
-          {
-            "key": "label",
-            "schema": {
-              "value": {
-                "generatedName": "DocumentsUploadRequestLabel",
-                "schema": {
-                  "minLength": 1,
-                  "type": "string"
-                },
-                "description": "A human-friendly name for this document. Typically the filename.",
-                "groupName": [],
-                "type": "primitive"
-              },
-              "type": "json"
-            }
-          },
-          {
-            "key": "contents",
-            "schema": {
-              "isOptional": false,
-              "isArray": false,
-              "type": "file"
```

Comment:
@amckinney this will be a break for vellum and we won't do file uploads for their endpoint

Reply:
Great call, fixed with the priority order you suggested.